applications/ConvectionDiffusionApplication/python_scripts/python_solvers_wrapper_convection_diffusion.py

```diff
@@ -36,6 +36,8 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
         # Steady solver
         elif (solver_type == "stationary" or solver_type == "Stationary"):
             solver_module_name = "convection_diffusion_stationary_solver"
+        elif (solver_type == "stationary_matrix"):
```

Comment:
This is not necessary, you could also call the module by providing the folder/file_name in the .json. But depends on how you wanna have it.

Reply:
how would that work?

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/b33095dd55ad719d7b6e04aa69dee03be552c1bf/applications/StructuralMechanicsApplication/tests/test_custom_scipy_base_solver.py#L33
Well not the folder, but the python module.

Reply:
But probably, it is only available in the StructuralMechanicsApplication. The solver_wrapper_structrual.py checks if it finds a module with the respective name.

Reply:
Interesting. This mechanism is not implemented in ConvectionDiffusionApplication however. But good to know this exists (at least for StructMech).